techdocs/README_TECHDOCS.md

```diff
@@ -0,0 +1,120 @@
+site_description: Documentation for my project
+docs_dir: "docs"
+plugins:
+    - techdocs-core
+    - ezlinks
+markdown_extensions:
+    - markdown_inline_mermaid
+    - md_in_html
+    -   mkpatcher:
+            location: patcher.py
+nav:
+    - Introduction: index.md
+    - Create an account: account.md
+    - Setup environment: setup.md
+    - Deploy locally: local.md
+    - Deploy to OpenShift: openshift.md
+```
+
+## Notes
+
+Note that MkDocs uses `mkdocs.yml`, not `mkdocs.yaml`, although both appear to work.
+Refer to the [mkdocs user guide](https://www.mkdocs.org/user-guide/configuration/) for more details.
+
+## Previewing documentation locally
+
+You will likely want to preview your documentation on your computer. Follow [these instructions](https://github.com/bcgov/devhub-techdocs-publish/blob/main/docs/index.md#how-to-use-the-docker-image-to-preview-content-locally) to do so.
+
+## Building and publishing your documentation
+
+The [GitHub Action workflow file](.github/workflows/publish.yaml) can validate, build (produce browser-friendly HTML from Markdown) and publish your documentation to DevHub automatically.  Initially, it is set up just to validate and build your documentation. 
+
+The portion of the workflow file that publishes your documentation is commented out because it requires addition set up with the Developer Experience team. When you'd like to have your documentation published in DevHub, [contact the Developer Experience Team](mailto:developer.experience@gov.bc.ca) who will work with you to set things up. 
+
+### The publishing workflow
+
+DevHub has a "preview" environment and a production environment, and once the Developer Experience team has you set up, and you've uncommented the publishing sections in the [workflow file](.github/workflows/publish.yaml) you'll be in control of how and when your content is published to each. You'll be able to control this by modifying the [workflow file](.github/workflows/publish.yaml) to suit your process and team.
+
+The configuration provided in the [workflow file](.github/workflows/publish.yaml) behaves as follows: 
+
+* A pull request to your repo will build and publish the code to the [DevHub preview](https://dev.developer.gov.bc.ca) site. This is a development site where you can preview your changes.
+* A release in your repository will publish the code to the [production DevHub website](http://developer.gov.bc.ca).
+
+## Getting help or reporting an issue
+
+To report bugs/issues/features requests, please email the Developer Experience team at [Developer.Experience@gov.bc.ca](mailto:Developer.Experience@gov.bc.ca)
+
+The full documentation for MkDocs (which is powering the DevHub TechDocs function) is at the [MkDocs website](https://www.mkdocs.org/user-guide/writing-your-docs/). Note that not all content on the site will be applicable to DevHub's use of MkDocs.
+
+## Contributing
+If you would like to contribute, please see our [contributing](CONTRIBUTING.md) guidelines.
```

Comment:
We should update this sentence because we do not have a contributing document

Reply:
updated in newest commit - [f74d835](https://github.com/bcgov/citz-imb-sso-express/pull/70/commits/f74d835b112e0adb23c3461aa43e6929f6802bbc)